### PR TITLE
Unmute testDeleteActionDeletesSearchableSnapshot

### PR DESCRIPTION
**Pull request by andreidan**
Unmuting this test as https://github.com/elastic/elasticsearch/pull/54433 was closed and I suspect
https://github.com/elastic/elasticsearch/pull/68309 was the cause.

### CODE REVIEW

**Review by dakrone — APPROVED** [review]
LGTM assuming CI is happy :)

**Comment by mark-vieira** [comment]
CI is not happy.

I reverted this since it failed as soon as merged.
https://gradle-enterprise.elastic.co/s/j7ia42wdhw53c/tests/:x-pack:plugin:ilm:qa:multi-node:javaRestTest/org.elasticsearch.xpack.ilm.actions.SearchableSnapshotActionIT/testDeleteActionDeletesSearchableSnapshot#1

**Pull request merged**
